packages/react/src/recipes/SolutionTemplates/SolutionPage/SolutionPage.tsx

```diff
@@ -0,0 +1,865 @@
+import lightHeroBg from '../../../fixtures/images/background-light-collaboration.webp'
+import darkHeroBg from '../../../fixtures/images/background-dark-collaboration.webp'
+import ciCdRenderImage from '../../../fixtures/images/ci-cd-render-ui.png'
+
+import clsx from 'clsx'
+
+import styles from './SolutionPage.module.css'
+import parallaxStyles from '../parallax.module.css'
+
+import {ColorModesEnum, ThemeProvider} from '../../../ThemeProvider'
+
+type SolutionPageProps = {
+  variant: 'size' | 'industry' | 'use-case'
+  gridOverlay?: boolean
+  colorMode?: ColorModesEnum.LIGHT | ColorModesEnum.DARK
+
+  heroLabel: string
+  heroTitle: string
+  heroDescription: string
+  heroCtaTextPrimary: string
+  heroCtaTextSecondary: string
+
+  introVariant: 'pillars' | 'editorial list'
+
+  [key: string]: unknown
+}
+
+export function SolutionPage({
+  variant,
+  gridOverlay = false,
+  colorMode = ColorModesEnum.LIGHT,
+  ...args
+}: SolutionPageProps) {
+  const [enableGridOverlay, setGridOverlay] = React.useState(gridOverlay)
+  const [isLightMode, setIsLightMode] = React.useState(colorMode === ColorModesEnum.LIGHT)
+  const selectedColorMode = isLightMode ? ColorModesEnum.LIGHT : ColorModesEnum.DARK
+
+  useEffect(() => {
+    setIsLightMode(colorMode === ColorModesEnum.LIGHT)
+  }, [colorMode])
+
+  const handleMode = e => {
+    e.preventDefault()
+    setIsLightMode(!isLightMode)
+  }
+
+  const handleKeyPress = useCallback(
+    (e: KeyboardEvent) => {
+      if (e.shiftKey && e.key === 'G') {
+        setGridOverlay(!enableGridOverlay)
+      }
+    },
+    [enableGridOverlay],
+  )
+
+  useEffect(() => {
+    document.addEventListener('keydown', handleKeyPress)
+    return () => {
+      document.removeEventListener('keydown', handleKeyPress)
+    }
+  }, [enableGridOverlay, handleKeyPress])
+
+  return (
+    <ThemeProvider
+      colorMode={selectedColorMode}
+      style={{
+        backgroundColor: 'var(--brand-color-canvas-default)',
+      }}
+      {...args}
+    >
+      <SubdomainNavBar title="" fixed={false}>
+        <SubdomainNavBar.SecondaryAction
+          aria-label={isLightMode ? 'Switch to dark mode' : 'Switch to light mode'}
+          href="#"
+          onClick={handleMode}
+          // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+          // @ts-ignore
+          variant="invisible"
+        >
+          {isLightMode ? <MoonIcon size={24} /> : <SunIcon size={24} />}
+        </SubdomainNavBar.SecondaryAction>
+      </SubdomainNavBar>
+      <AnimationProvider runOnce visibilityOptions={0.2}>
+        <section>
+          {(variant === 'industry' || variant === 'use-case') && (
+            <Box className={styles.hideHorizontalOverflow}>
+              <Grid enableOverlay={enableGridOverlay} className={clsx({[styles.heroVideoGlow]: args.heroVideo})}>
+                <Grid.Column>
+                  <Box marginBlockStart={20}>
+                    <Breadcrumbs>
```

Comment:
The breadcrumbs might need their spacing looking at, it looks like they're disappearing behind the nav bar. I ran into something similar with the parallax hero and applied a bit of margin-top to them to make sure they're visible iirc

Reply:
The breadcrumbs only appear in the industry and use-case variants, not size (which is the one that has parallax enabled). You can see the conditional for it on [line 122](https://github.com/primer/brand/pull/623/files/1e9c3c803987fb1ad7f45725195a237ba85146e4#diff-e4e4c13b20e60d9cd21c463988e14fd7c99bf606e5c2667c9890dabf8fb913fcR122). 

Should appear fine in the other templates though?
![Screenshot 2024-07-10 at 14 33 19](https://github.com/user-attachments/assets/09b01650-1d21-4943-9187-32605f933dcd)

Reply:
Uploading Untitled.mov…

Reply:
This is what I'm seeing (top-left)

https://github.com/user-attachments/assets/eda036fb-0152-4c6a-b0b6-a505b409931a

Reply:
Ah! you're right, thanks for spotting that 👍 . I've added the margin offset here, as it should appear here but not on the overview page.